includes/User.php

```diff
@@ -2416,11 +2417,183 @@ public function getIntOption( $oname, $defaultOverride=0 ) {
 		return intval( $val );
 	}
 
+
+	/**
+	 * Get a preference local to this wikia.
+	 *
+	 * @param string $preference the preference name
+	 * @param int $cityId the city id
+	 * @param string $sep the separator between the name and the city id
+	 * @return string
+	 */
+	public function getLocalPreference($pref, $cityId = null, $sep = "-") {
+		global $wgCityId;
+		if (!isset($cityId)) {
+			$cityId = $wgCityId;
+		}
```

Comment:
@garthwebb it's magic! Actually that work is happening over [here](https://github.com/Wikia/app/pull/7723). That branch was taken off of this one and will be merged in here before we merge to dev. So yes, when we are done something will actually get the preferences :).